Rebecca

Rebecl opened the pull request.

slahrichi committed:
ADD: Basic version of image extraction tool

slahrichi committed:
ADD: Generate random coordinates & Download images to disk

slahrichi committed:
ADD: Python script to download images from GEE

zcalhoun commented on imageExporter.py:
```diff
@@ -0,0 +1,166 @@
+# Not working yet
+logger = logging.getLogger('my_logger')
+logging.basicConfig(filename='file.log',
+                    filemode='a')
```
If this isn't working -- you might try changing the filemode to 'w' (see the modes [here](https://docs.python.org/3/library/functions.html#filemodes))

zcalhoun commented on imageExporter.py:
```diff
@@ -0,0 +1,166 @@
+    coords = pd.read_csv(filepath)
+    start_time_recorded = False
+
+    for i in tqdm(range(len(coords))):
```
I told Saad this, but it's cleaner code to use `for i, (lat, lon) in enumerate(coords)` -- that should get rid of 3 lines of code while making the code easier to read, too.

zcalhoun commented on imageExporter.py:
```diff
@@ -0,0 +1,166 @@
+    args = parser.parse_args()
+    if not os.path.exists(args.output_dir):
+        os.makedirs(args.output_dir)
+        pass
```
you can get rid of the pass here

zcalhoun commented on imageExporter.py:
```diff
@@ -0,0 +1,166 @@
+                                       output_dir=args.output_dir)
+    export_finish_time = time.time()
+    duration = export_finish_time - export_start_time
+    print(f'Export complete! Total duration: {duration} s')
```
remember to change this to use the logger

zcalhoun reviewed (commented):
Added some comments to the script about a few minor syntax fixes. Let's plan on looking at your code together this morning if you're still having issues.

Rebecl committed:
Add files via upload

Rebecl committed:
added logging

Rebecl committed:
Created using Colaboratory

Rebecl committed:
Add files via upload

Rebecl committed:
Delete Sampling_from_Cities_.ipynb

Rebecl committed:
Created using Colaboratory

Rebecl committed:
Delete ClimateEye.ipynb

Rebecl committed:
Delete imageExporter.py

Rebecl committed:
Delete imageExporter(wMP).py

Rebecl committed:
Created using Colaboratory

zcalhoun committed:
SwAV (#3)

The working version of the SwAV notebook included for future reference in case somebody wants to look at how to apply this model.

zcalhoun committed:
Added rejection sampling method. (#4)

This contains the notebook for looking at sampling methods and understanding rejection sampling. Saving this in the repository for future usage.

zcalhoun committed:
Rename Sampling_from_Cities_.ipynb to notebooks/Sampling_from_Cities_.ipynb

zcalhoun committed:
Delete World_Continents.shp

zcalhoun committed:
Delete World_Continents.shx

zcalhoun committed:
Delete worldcities.csv
